src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,245 @@
+                drivers.add(parseDriver(resultSet));
+            }
+            return drivers;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't get info about drivers in car with id "
+                    + id + ". ", throwable);
+        }
+    }
+
+    private Driver parseDriver(ResultSet resultSet) throws SQLException {
+        Driver driver = new Driver();
+        driver.setId(resultSet.getObject("driver_id", Long.class));
+        driver.setName(resultSet.getString("driver_name"));
+        driver.setLicenseNumber(resultSet.getString("driver_license_number"));
+        return driver;
+    }
+
+    public boolean deleteDrivers(Long id) {
+        String query = "UPDATE taxi.drivers "
+                + "SET taxi.drivers.is_deleted = TRUE "
+                + "WHERE taxi.drivers.id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement deleteDriversStatement = connection.prepareStatement(query)) {
+            deleteDriversStatement.setLong(1, id);
+            return deleteDriversStatement.executeUpdate() > 0;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't get car by id " + id, throwable);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "UPDATE taxi.cars SET taxi.cars.is_deleted = TRUE WHERE taxi.cars.id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement deleteCarStatement = connection.prepareStatement(query)) {
+            deleteCarStatement.setLong(1, id);
+            return deleteCarStatement.executeUpdate() > 0;
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't get car by id " + id, throwable);
+        }
+    }
+
+    @Override
+    public Car update(Car car) {
+        String query = "UPDATE taxi.cars "
+                + "SET taxi.cars.manufacturer_id = ?, taxi.cars.model = ? "
+                + "WHERE taxi.cars.id = ? AND taxi.cars.is_deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement updateCarStatement
+                        = connection.prepareStatement(query)) {
+            updateCarStatement.setLong(1, car.getManufacturer().getId());
+            updateCarStatement.setString(2, car.getModel());
+            updateCarStatement.setLong(3, car.getId());
+            updateCarStatement.executeUpdate();
+        } catch (SQLException throwable) {
+            throw new DataProcessingException("Couldn't update "
+                    + car + " in carsDB.", throwable);
+        }
+        deleteDrivers(car.getId());
+        createCarsAndDrivers(car);
+        return car;
+
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String query = "SELECT "
+                + "taxi.drivers.id AS driver_id, "
```

Comment:
same here, let's shorten this request to improve readability, also IMHO we don't need aliases here